core/src/trezor/_proto_messages.mako

```diff
@@ -30,12 +30,12 @@ optional_fields = [f for f in message.fields if f.optional]
 
 def member_type(field):
     if field.required:
-        return field.python_type
+        return f"\"{field.python_type}\""
```

Comment:
instead, put the quotes around the whole thing at the usage site. `list["SomeType"]` is equivalent to `"list[SomeType]"`

Speaking of, for firmware we're in python 3.10 mode so I'm wondering if we even need the quotes here at all. I didn't try that though.

Reply:
I tried leaving the quotes out however mypy (and also PyCharm) complains then - I'm not sure if mypy is still on old Python or if this isn't supported even in 3.10 though.